src/interpreter/main.cpp

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+#include "../include/VM.h"
+#include "../include/opcode.h"
+
+using namespace Opcode;
+
+int main() {
+    // 1, 1, ADD, 1, ADD, RETURN, HALT
+    unsigned char bytecodes[] = {ICONST_0, ICONST_1, ADD, PRINT, HALT};
```

Comment:
Might want to consider `std::array` here as well (although it's pretty marginal so maybe not important).